app/models/repository.rb

```diff
@@ -328,4 +329,8 @@ def sorted_tags
   def repository_host
     @repository_host ||= RepositoryHost.const_get(host_type.capitalize).new(self)
   end
+
+  def gather_maintenance_stats
+    CriticalRepositoryMaintenanceStatWorker.perform_async(id)
```

Comment:
Looks like you didn't add the worker?

Reply:
Yeah I had that as a separate story, but I can just commit the workers in this PR.